docs/source/models/intervention_models/mncnh_pregnancy/oral_iron_antenatal/oral_iron_antenatal.rst

```diff
@@ -0,0 +1,606 @@
+  We will not be modeling Balanced Energy Protein (BEP) supplementation in this simulation, nor the impact BEP and MMS have on child growth failure. 
+
+Maternal supplementataion during the prenatal period is a critical intervention to support the increased nutritional demands due to the mother's changing physiology and fetal growth. 
+Maternal supplementation in low and middle income country settings are often accessed through antenatal care (ANC) services and early initiation of supplementation has been associated with improved outcomes. 
+There are various maternal supplementation regimens that provide varying combinations of micro- and macro-nutrients and therefore have varying impacts on both maternal and child health. 
+The two major classes of maternal supplementation regimens that we will be modeling are outlined in the table below.
+
+- **Iron and Folic Acid (IFA):** IFA affects outcomes including maternal anemia (which is a risk factor for maternal mortality) and infant birthweight (a risk factor for infant mortality). Additionally, folic acid deficiency is associated with haematological consequences and congenital malformations, however the effect on congenital malformations is sensitive to timing of administration relative to conception. The latest cochrane review of IFA trials was performed by [Pena-Rosas-et-al-2015]_; notably, as IFA is considered the current standard of care, there are few studies that compare IFA to placebo or no treatment.
+
+- **Multiple Micronutrient Supplementation (MMS):** MMS is a combination of micronutrients that includes iron and folic acid as well as other nutrients such as vitamin A and zinc. The latest cochrane review on MMS (containing IFA) relative to IFA alone was performed by [Keats-et-al-2019-maternal-supplementation]_.
+
+In August of 2020, the :download:`World Health Organization updated their recommendation <who_guidance_positive_pregnancy_mms_2020.pdf>` on nutritional interventions during pregnancy to include MMS that includes iron and folic acid as part of routine antenatal care. 
+Prior to this update, they recommended only IFA. 
+
+.. note:: 
+
+  According to the :download:`2016 WHO recommendations <who_guidance_positive_pregnancy_2016.pdf>`: Undernourishment is usually defined by a low BMI (i.e. being underweight). 
+  For adults, a 20–39% prevalence of underweight women is considered a high prevalence of underweight and 40% or higher is considered a very high prevalence (46). 
+  Mid-upper arm circumference (MUAC) may also be useful to identify protein–energy malnutrition in individual pregnant women and to determine its prevalence in this population (31). 
+  However, the optimal cut-off points may need to be determined for individual countries based on context-specific cost–benefit analyses (31).
+
+IFA is widely used as a prenatal supplement in most areas of the world and is recommended by the WHO as part of routine antenatal care, although coverage and adherence levels vary. 
```

Comment:
```suggestion
IFA is widely used as a prenatal supplement in most areas of the world and has been recommended by the WHO as part of routine antenatal care for quite some time, although coverage and adherence levels vary. 
```

Since both are recommended, maybe emphasizing this is good? But I'm not sure it's necessary to do so